test/verify/run-tests

```diff
@@ -31,15 +32,16 @@ def check_valid(filename):
 
 def run(opts):
     # Now actually load the tests, any modules that start with "check-*"
-    loader = unittest.TestLoader()
+    test_loader = unittest.TestLoader()
     suite = unittest.TestSuite()
     for filename in glob.glob(os.path.join(os.path.dirname(__file__), "check-*")):
         name = check_valid(filename)
         if not name or not os.path.isfile(filename):
             continue
-        with open(filename, 'rb') as fp:
-            module = imp.load_module(name, fp, filename, ("", "rb", imp.PY_SOURCE))
-            suite.addTest(loader.loadTestsFromModule(module))
+        loader = importlib.machinery.SourceFileLoader(name, filename)
+        module = importlib.util.module_from_spec(importlib.util.spec_from_loader(loader.name, loader))
```

Comment:
Calling the `importlib.util` stuff from outside of a loader implementation seems pretty odd to me, particularly `spec_from_loader()`.  Does `spec_from_file_location()` work there, by chance?

Reply:
It does not.  Probably caused by the fact that our tests are not named like normal Python modules...